src/Propulsion.EventStore/Infrastructure.fs

```diff
@@ -1,4 +1,8 @@
-﻿namespace Propulsion.EventStore
+﻿#if EVENTSORE
```

Comment:
typo (also I'd just make this the else bit - i.e. ES can be default case

Reply:
Fixed.